dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers/get-utilisation-details.test.ts

```diff
@@ -0,0 +1,81 @@
+import { CURRENCY, FeeRecordEntityMockBuilder } from '@ukef/dtfs2-common';
+import { when } from 'jest-when';
+import { getUtilisationDetails } from './get-utilisation-details';
+import { aTfmFacility, aUtilisationReport } from '../../../../../../test-helpers';
+import { TfmFacilitiesRepo } from '../../../../../repositories/tfm-facilities-repo';
+import { NotFoundError } from '../../../../../errors';
+import { mapToFeeRecordUtilisation } from './map-to-fee-record-utilisation';
+import { FeeRecordUtilisation } from '../../../../../types/fee-records';
+
+jest.mock('../../../../../repositories/tfm-facilities-repo');
+jest.mock('../../../../../helpers');
+jest.mock('./map-to-fee-record-utilisation');
+
+describe('get-utilisation-details', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('getUtilisationDetails', () => {
+    it('throws if no tfm facility is found for one of the fee records', async () => {
```

Comment:
could have a describe here.  `describe('if no tfm facility is found.....`
also all it statements should start with should it('should...

Reply:
Will add